libraries/SITL/SITL.h

```diff
@@ -98,6 +98,8 @@ class SITL
     AP_Int16  mag_delay; // magnetometer data delay in ms
     AP_Int16  wind_delay; // windspeed data delay in ms
 
+    AP_Int16 gps_spd_err; // reported GPS speed accuracy in cm/s
```

Comment:
I'll let @priseborough comment on this. I think the controller variables have quite different names though.